### ISSUE TITLE
IntuneASRApplicationControlPolicyWindows10: New resource proposal

### ISSUE THREAD

**Issue by ricmestre** [microsoft/Microsoft365DSC#3029]
### Description

This resource would be used to export/apply Attack Surface Reduction policies for Application Control

### Proposed properties

All

### Special considerations or limitations

The cmdlets to deal with this resource in Graph are {New,Get,Update}-MgDeviceManagementIntent{Setting} and are currently already used in resource MSFT_IntuneASRRulesPolicyWindows10

**Comment by William-Francillette** [comment]
@ricmestre 
this resource is already available as IntuneApplicationControlPolicyWindows10

**Comment by ricmestre** [comment]
Correct, looking at the names of the resources I saw IntuneASRRulesPolicyWindows10 and IntuneAttackSurfaceReductionRulesPolicyWindows10ConfigManager, so this one without ASR in the name I assumed it was for something else, mea culpa. This one can be closed, thank you!
